integrations/spark/openhouse-spark-itest/build.gradle

```diff
@@ -41,10 +43,18 @@ task statementTest(type: Test) {
   }
 }
 
+task catalogTest(type: Test) {
```

Comment:
why is this needed?

Reply:
To order the tests to run after each other. Each of the sparkSessions from statementTest, CatalogTest, SparkBase test conflict with each other.